about.css

```diff
@@ -370,7 +373,7 @@ body {
 footer {
   width: 375px;
   height: 200px;
-  background-color: #d3d3d3;
+  background-color: #808080;
   position: absolute;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219131368-636c5978-2fd0-444c-bdfd-ef2c082f41c4.png)

- [ ] The previous reviewer wanted you to use the dark background shown in the image above :arrow_up: but the color you are using is `#808080` you are supposed to use `#2e2e2e` :blush: